src/modules/Dashboard/api/api.types.ts

```diff
@@ -95,6 +95,10 @@ export const enum TimerType {
   Idle = 'IDLE',
 }
 
+export const enum PeriodicityAlways {
+  Always = 'ALWAYS',
+}
+
```

Comment:
Why did you use enum instead of primitive const?

Reply:
done